src/action-object.ts

```diff
@@ -3,20 +3,33 @@ import { requireProperty, ID } from './utils'
 export enum ActionTypeV1 {
   GET = 'GET',
   SET = 'SET',
-  ACTION = 'ACTION'
+  ACTION = 'ACTION',
+  INIT = 'INIT'
 }
 
 export enum CommunicationMethodV1 {
-  JSONRPC = 'JSONRPC',
+  HTTP = 'HTTP',
   SERIAL = 'SERIAL'
 }
 
+export enum ProtocolV1 {
+  JSONRPC = 'JSONRPC',
+  CLI = 'CLI'
+}
+
+export type CommunicationDataV1 = {
+  commMethod: CommunicationMethodV1
+  username?: string
+  password?: string
+}
+
```

Comment:
I'd move the protocol to be under the CommunicationData object rather than it being at the root of ActionObjectInformation.

I feel like the protocol is only related to communication with the radio rather than information about the action to be performed